identity-wallet/src/state/actions/load_dev_profile.rs

```diff
@@ -1,13 +1,51 @@
 use crate::reducer;
+use ts_rs::TS;
 use crate::state::actions::ActionTrait;
 use crate::state::actions::Reducer;
 use crate::state::reducers::dev_mode::load_dev_profile;
 
-#[derive(serde::Serialize, serde::Deserialize, Debug, Clone)]
-pub struct LoadDevProfile;
+#[derive(serde::Serialize, serde::Deserialize, Debug, TS, Clone, PartialEq, Eq)]
+#[ts(export, export_to = "bindings/dev/ProfileType.ts")]
+pub enum ProfileType {
+    /// Dev profile which preloads data
+    Ferris,
 
-#[typetag::serde(name = "[DEV] Load profile")]
-impl ActionTrait for LoadDevProfile {
+    /// Dev profile which recreates steps
+    Dragon,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Debug, TS, Clone, PartialEq, Eq, PartialOrd)]
+#[ts(export, export_to = "bindings/dev/ProfileSteps.ts")]
+pub enum ProfileSteps {
+    /// Step 1
+    CreateProfile,
+    /// Step 2
+    AddCredentials,
+    /// Step 3
+    AcceptCredentials,
+    /// Step 4
+    AddConnection,
+    /// Step 5
+    AcceptConnection,
+    /// Step 6
+    AddPresentation,
+    /// Step 7
+    ShareCredentails,
+    /// Step 8
+    AddFutureEngineer,
+    /// Step 9
+    CompleteFlow,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Debug, TS, Clone)]
+#[ts(export, export_to = "bindings/actions/DevProfile.ts")]
+pub struct DevProfile {
+    pub profile: ProfileType,
+    pub execute_steps: Option<ProfileSteps>,
```

Comment:
How can multiple steps be executed with a single `Option<enum>`?

Reply:
Because the enum represent multiple steps, it all builds on the previous steps.

Reply:
I know, but what I'm saying is that `Option<ProfileSteps>` can only hold a single step at a time, right?

Reply:
Sure I will change it, I was questioning too if this should be plural or not